.github/scripts/pr_body_check.py

```
@@ -0,0 +1,64 @@
+
+def main():
+    pr_body = os.environ.get('PR_BODY', '')
+    if not pr_body:
+        warn('PR description is missing! Please provide a valid PR description.')
+        return
+
+    template_path = os.path.join(os.environ.get('GITHUB_WORKSPACE', '.'), '.github', 'pull_request_template.md')
+    try:
+        with open(template_path, encoding='utf-8') as f:
+            template_content = f.read().strip()
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)